test/e2e-common.sh

```diff
@@ -255,9 +255,36 @@ function install_ambassador() {
 function install_knative_serving_standard() {
   readonly INSTALL_CERT_MANAGER_YAML="./third_party/cert-manager-${CERT_MANAGER_VERSION}/cert-manager.yaml"
 
-  # If we need to build from source, then kick that off first.
+  echo ">> Installing Knative namespace and CRD"
   if [[ -z "$1" ]]; then
+    # If we need to build from source, then kick that off first.
     build_knative_from_source
+
+    echo "Namespace YAML: ${SERVING_NAMESPACE_YAML}"
+    echo "CRD YAML: ${SERVING_CRD_YAML}"
+    kubectl apply \
+	    -f "${SERVING_NAMESPACE_YAML}" \
+	    -f "${SERVING_CRD_YAML}" || return 1
+    UNINSTALL_LIST+=( "${SERVING_NAMESPACE_YAML}" "${SERVING_CRD_YAML}" )
+  else
+    echo "Knative YAML: ${1}"
+    ko apply -f "${1}" --selector=knative.dev/crd-install=true || return 1
+    ko apply -f "${1}" --selector=knative.dev/namespace-install=true \
+       || ko apply -f "${1}" --selector=istio-injection=enabled `#TODO: Remove this after v0.12 is released.` \
```

Comment:
You aren't using the new label?

Reply:
I don't think the new label will work for this path anyways because I think that this flow is generally taken when installing a *prior* release (maybe you hit a problem with the upgrade test, which is why this is using the injection label?).

I wonder if we could drop the label from namespace if we simply added here:
```
kubectl apply -f - <<EOF
apiVersion: v1
kind: Namespace
metadata:
  name: knative-system
  labels:
    istio-injection=enabled
EOF
```

But probably not because applying the namespace without it later (before the deployments are created) would negate it 🙄

Reply:
But you could probably unblock by creating the namespace without the label and leaving the label in our yaml intact 😢

Reply:
Thank you.

> maybe you hit a problem with the upgrade test, which is why this is using the injection label?

Yes, you are correct. That is what I experienced today :sweat_smile:
Now, I agreed that creating namesapce, so updated.